src/raftstore/store/worker/apply.rs

```diff
@@ -564,7 +551,6 @@ impl ApplyDelegate {
         // others will be saved as a normal entry with no data, so we must re-propose these
         // commands again.
         apply_ctx.committed_count += committed_entries.len();
```

Comment:
Seems `committed_count` is updated twice?

Reply:
@zhangjinpeng1987 Was this resolved?

Reply:
Done